cloudinary/cloudinary_wordpress
# Update the readme

Updates the readme information for the plugin

## Approach

- n/a

## QA notes

- You can't see these updates before launching them, but you can use the following to validate and preview them:
  - https://wordpress.org/plugins/developers/readme-validator/
  - https://wpreadme.com/
